src/Altinn.Notifications.Core/Extensions/ServiceCollectionExtensions.cs

```diff
@@ -19,7 +17,7 @@ public static class ServiceCollectionExtensions
     /// </summary>
     /// <param name="services">service collection.</param>
     /// <param name="config">the configuration collection</param>
-    public static IServiceCollection AddCoreServices(this IServiceCollection services, IConfiguration config)
+    public static void AddCoreServices(this IServiceCollection services, IConfiguration config)
```

Comment:
why change this to void?

Reply:
To have the same notation for adding health check and service. 
Don't see an explicit need to the serviceCollection.

Reply:
I'm not always too fond of dot chaining, but it's still common to support it.